src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1745,8 +1911,12 @@ void AutomationInstrumentClipView::auditionPadAction(int32_t velocity, int32_t y
 					instrumentClipView.sendAuditionNote(false, yDisplay, 64, 0);
 				}
 			}
+			display->cancelPopup();
 			instrumentClipView.someAuditioningHasEnded(true);
 			actionLogger.closeAction(ACTION_NOTEROW_ROTATE);
```

Comment:
added to re-render display after auditioning has ended. used if you're still holding a pad to re-display the parameter value or otherwise just re-display the parameter name